tables/cost_and_usage_focus/cost_and_usage_focus_extractor.go

```diff
@@ -122,12 +122,12 @@ func (log *CostUsageFocus) mapValues(row map[string]string) error {
 					}
 
 				case reflect.Float64:
-					if floatVal, err := strconv.ParseFloat(value, 64); err == nil {
+					if floatVal, err := strconv.ParseFloat(value, 64); err == nil  && floatVal > 0{
```

Comment:
@ParthaI When is `floatVal` = 0? If it ever equals 0 because that's what it's in the actual Cost and Usage report data, then we should set that column value as 0.

Reply:
Ah, that makes sense. In the CSV report, I noticed the value is shown as `0.0` instead of being blank or null. I've pushed the changes accordingly.